src/Http/Routing/ref/Microsoft.AspNetCore.Routing.Manual.cs

```diff
@@ -199,6 +199,8 @@ internal static partial class FastPathTokenizer
     internal partial class DfaMatcherBuilder : Microsoft.AspNetCore.Routing.Matching.MatcherBuilder
     {
         public DfaMatcherBuilder(Microsoft.Extensions.Logging.ILoggerFactory loggerFactory, Microsoft.AspNetCore.Routing.ParameterPolicyFactory parameterPolicyFactory, Microsoft.AspNetCore.Routing.Matching.EndpointSelector selector, System.Collections.Generic.IEnumerable<Microsoft.AspNetCore.Routing.MatcherPolicy> policies) { }
+        internal EndpointComparer Comparer { get; }
+        internal bool UseCorrectCatchAllBehavior { get; set; }
```

Comment:
Note, this is an internal type. In servicing our internal types are in the ref-assemblies, and so we have to do stuff like this.